tests/wpt/README.md

```diff
@@ -128,6 +128,8 @@ be run in the following manner like any other WPT test:
     ./mach test-wpt tests/wpt/path/to/new/test.html
     ./mach test-wpt tests/wpt/path/to/new/reftest.html
 
+The tests are written using [testharness.js](https://github.com/w3c/testharness.js/). Documentation can be found [here](http://testthewebforward.org/docs/testharness-library.html).
```

Comment:
@Ms2ger Perhaps it is more apt for your link to be at the top of the page to serve as a primer for writing tests in general. 
We can get more specific by setting a link for reftests after its explanation ends and the same for testharness.js tests.